### ISSUE TITLE
radvd not working if more than 3 ipv6 dns servers are set

### ISSUE THREAD

**Issue by je4** [opnsense/core#6448]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Describe the bug**

A clear and concise description of what the bug is, including last known working version (if any).
After adding too many (more than 3?) IPv6 DNS server, raddvd is not able to start. 

**To Reproduce**

1. Go to `System: Settings: General`
2. Add more than 3  dns server with ipv6 address (gateway seems to be irrelevant)
3. Save 
4. Go to `System: Diagnostics: Services`
5. See error: radvd not running any more

**Expected behavior**

Since there's no limitation for IPv6 DNS Servers on the interface i would expect, that radvd is running as usual.

**Describe alternatives you considered**

**Screenshots**

`System: Settings: General`

![image](https://user-images.githubusercontent.com/8255588/227986292-eb26d997-3171-4efb-83f1-0f26a928a949.png)

`System: Diagnostics: Services`

![image](https://user-images.githubusercontent.com/8255588/227986597-4e5425c7-461c-4438-ae22-03a6d1cb2eee.png)

**Additional context**

### radvd.conf
```
RDNSS 2001:1620:2777:1::10 2001:1620:2777:2::20 2001:4860:4860::8888 2620:fe::fe { };
```
### error message 
```
 radvd (67600): too many addresses in RDNSS section
```

#6317 seems to be similar

**Environment**

OPNsense 23.1.4_1-amd64
FreeBSD 13.1-RELEASE-p7
OpenSSL 1.1.1t 7 Feb 2023
Intel(R) Celeron(R) CPU J1900 @ 1.99GHz (4 cores, 4 threads)

**Comment by fichtner** [comment]
For reference:

https://github.com/radvd-project/radvd/blob/f67335b5335b6ed5ca68d6fa71c08cccb4f3a629/gram.y#L807

Seems to be the only error of this kind. Patch is e1da915, maybe you can confirm?

    # opnsense-patch e1da915


Cheers,
Franco

**Comment by fichtner** [comment]
To be shipped in 23.1.6.

**Comment by Leseratte10** [comment]
Radvd 2.20 has just been released and the restriction to limit to three RDNSS entries has been lifted (it's now 127). Can the radvd in opnsense be updated to 2.20 and this restriction be removed again?
